vrc-get-gui/locales/en.json5

```
@@ -153,5 +158,9 @@
     "opening unity...": "Opening Unity...",
     "selecting file or folder": "Selecting file or folder",
     "please select a file or folder": "Please select a file or folder",
+    "backup project": "Backup Project",
+    "creating a backup...": "Creating a backup...",
+    "backup cancelled": "Backup cancelled",
```

```suggestion
    "backup canceled": "Backup canceled",
```
アメリカ英語 vs イギリス英語っぽい? (他だとactionsとかには紛れてそうです)